frontend/src/components/AccountSentTable.vue

```diff
@@ -1,97 +1,127 @@
 <template>
-  <q-table
-    :grid="$q.screen.xs"
-    card-container-class="col q-col-gutter-md"
-    :columns="mainTableColumns"
-    :rows="formattedSendMetadata"
-    :no-data-label="$t('AccountSendTable.account-empty')"
-    :pagination="paginationConfig"
-    row-key="dateSentUnix"
-    :title="$t('AccountSendTable.sent-funds')"
-  >
-    <template v-slot:top-left="props">
-      <div class="flex">
-        <div class="q-table__title" :props="props">{{ $t('AccountSendTable.sent-funds') }}</div>
-        <base-tooltip class="q-ml-sm self-center" size="xs" icon="fas fa-question-circle">
-          {{ $t('AccountSendTable.storage-description') }}
-        </base-tooltip>
-      </div>
-    </template>
-    <!-- Card Layout for grid option -->
-    <template v-slot:item="props">
-      <div :key="props.row.dateSentUnix" class="col-12">
-        <q-card class="card-border cursor-pointer q-pa-md col justify-center items-center">
-          <q-card-section class="row justify-center items-center">
-            <img class="q-mr-md" :src="props.row.tokenLogo" style="width: 1.2rem" />
-            <div class="text-primary text-h6 header-black q-pb-none">
-              {{ props.row.amount }} {{ props.row.tokenSymbol }}
-            </div>
-          </q-card-section>
-          <div class="row justify-between items-center">
-            <div>
-              <span class="q-mr-xs">{{ $t('AccountSendTable.receiver') }}</span>
-            </div>
-            <div>
-              <div @click="copyAddress(props.row.address, provider)" class="cursor-pointer copy-icon-parent">
-                <span>{{ props.row.addressShortened }}</span>
-                <q-icon color="primary" class="q-ml-sm" name="far fa-copy" />
+  <div>
+    <div class="text-caption q-mb-sm">
+      {{ $t('AccountSentTable.stored-on-device') }}.
+      <router-link
+        class="cursor-pointer hyperlink"
```

Comment:
_(arbitrary comment location to start a thread)_

If the count key is deleted, but the values array key is still present we can't handle that and throw an error, and get stuck on the loading screen. I think we should log a warning with `window.logger.warn` and clear the existing values, since we won't be able to decrypt them

(I did verify the opposite works as expected, e.g. clearing the send values array key but keeping the count generates a new count)


<img width="1351" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/17163988/908fb996-ab0e-48bc-a62f-ff2ef1be0183">

Reply:
Handled and the tests have been updated to handle null

https://github.com/ScopeLift/umbra-protocol/pull/494/commits/41e41d5e97ea036682dd17235f7367ce29551490

Reply:
Yes, I agree removeItem is better semantically. I guess in my mind I was always think it was resetting to 0 items rather then removing the key. But yes, it doesn't matter either way.

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/494/commits/3756a1b286e07af390c82b865d0a33b6ad57eeb3